9.4/paper.tex

```diff
@@ -563,7 +563,7 @@ \subsection{Advances in matrix-free infrastructure}\label{sec:mf}
 contexts. Examples are simulations with sharp interfaces (e.g., two-phase flow
 or shock capturing), where one needs to treat cells that are ``cut'' by
 the interface in a special way. A challenge is that cell batches
-might contain cut or non-cut cells, making vectorization of these operations potentially more complicated. Previous functionality has provided the option of masking certain cells in cell batches, which works well if
+might contain cut or non-cut cells, making the vectorization of these operations potentially more complicated. Previous functionality has provided the option of masking certain cells in cell batches, which works well if
```

Comment:
Both of this sounds correct to me after 20 years speaking mostly English. But it doesn't hurt.